services/movie.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet[Movie]:
+    if genres_ids and actors_ids:
+        return (Movie.objects
+                .filter(actors__id__in=actors_ids)
+                .filter(genres__id__in=genres_ids))
+
+    if genres_ids:
+        return Movie.objects.filter(genres__id__in=genres_ids)
+
+    if actors_ids:
+        return Movie.objects.filter(actors__id__in=actors_ids)
+
+    return Movie.objects.all()
```

Comment:
instead of making return everywhere you can create a variable queryset that initially contains all movies.
then you add ifs that filters this queryset(and updates it but doesn't return queryset). In this way python'll reach each if statement and will return what you need